android/app/src/main/java/com/chorestory/module/AuthModule.java

```diff
@@ -1,7 +1,5 @@
 package com.chorestory.module;
 
-import android.app.Application;
```

Comment:
TODO: Import optimization should be done by ide on whole project level sometime.